blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -439,7 +443,7 @@ export const actions = {
                                 return { ...data,
                                     id: event.jenkins_object_id, // make sure the runId is set so we can find it later
                                     state: 'RUNNING', // This is a horrible hack due to issues with QUEUED status
-                                    result: event.job_run_status,
+                                    result: 'UNKNOWN',
```

Comment:
Why are we changing this Vs using the state from the event? The state is not unknown afaik (it's [`QUEUED`](https://github.com/jenkinsci/pubsub-light-module/blob/master/src/main/java/org/jenkins/pubsub/listeners/SyncQueueListener.java#L82), I think).